addons/purchase/report/purchase_order_templates.xml

```diff
@@ -88,7 +88,7 @@
                                     <span class="text-align-bottom"><span t-field="line.discount"/>%</span>
                                 </td>
                                 <td class="text-end">
-                                    <span t-esc="', '.join(map(lambda x: x.description or x.name, line.taxes_id))"/>
```

Comment:
It seems like the `x.description` here was changed in this PR https://github.com/odoo/odoo/pull/160342 when it shouldn't have been. Can you please change it back to `x.invoice_label`? Or even better, back to the original `join([(tax.invoice_label or tax.name) for tax in line.taxes_id])`?

Reply:
hello, I'll update it